funnel/views/profile.py

```diff
@@ -51,6 +51,7 @@ def view(self):
             .limit(1)
             .first()
         )
+        unscheduled_projects = projects.filter(Project.schedule_state.PUBLISHED_WITHOUT_SESSIONS).all()
```

Comment:
This seems like it hasn't passed through `pre-commit`. Is it configured for you @qurbat?